### DISCUSSION TITLE
前台页面加载失败，装了好多遍没发现问题。这样搞很难用呀。

### DISCUSSION THREAD

**Discussion by lxz902** [hhyo/Archery#2634] [Q&A]
![image](https://github.com/hhyo/Archery/assets/37208299/b13c1049-e30e-4d8f-beb6-443908dc2ec1)

手动安装1.8.5，本地测试可以。用脚本启动就这样了。我DEBUG=Ture 也不行。
谁能给看看，我看也没有用nginx，就是gunicorn启动。
![image](https://github.com/hhyo/Archery/assets/37208299/b8045492-2870-4b0a-a2fd-cbd285afdde2)

**Comment by LeoQuote** [comment]
这边支持不了那么老的版本，请按手册内容部署，手动部署需要对django 和项目都有一定了解，如果遇到问题还请多探索一下。
本项目为开源项目，如果觉得比较难用，欢迎pr
